pulpcore/pulpcore/app/viewsets/custom_filters.py

```diff
@@ -46,3 +46,23 @@ def filter(self, qs, value):
 
         key = "{}__pk".format(self.name)
         return qs.filter(**{key: pk})
+
+
+class HelpTextFilterSet(filterset.FilterSet):
```

Comment:
Which FilterSets will this replace? If its all of them, it might be better to call this BaseFilterSet or something.